enterprise/api/v1/serializers.py

```diff
@@ -599,6 +599,7 @@ class EnterpriseGroupMembershipSerializer(serializers.ModelSerializer):
     learner_id = serializers.IntegerField(source='enterprise_customer_user.id', allow_null=True)
     pending_learner_id = serializers.IntegerField(source='pending_enterprise_customer_user.id', allow_null=True)
     enterprise_group_membership_uuid = serializers.UUIDField(source='uuid', allow_null=True, read_only=True)
+    enterprise_customer = EnterpriseCustomerSerializer(source='group.enterprise_customer', read_only=True)
```

Comment:
Discussed offline with @katrinan029 and I think we don't need the customer details in the membership serializer. Too much duplicated data and the information is easily fetched by the customer endpoint